src/metric-engine/src/test_util.rs

```diff
@@ -209,6 +174,58 @@ pub fn alter_logical_region_add_tag_columns(new_tags: &[&str]) -> RegionAlterReq
     }
 }
 
+/// Generate a [RegionCreateRequest] for logical region.
+/// Only need to specify tag column's name
+pub fn create_logical_region_request(
+    tags: &[&str],
+    physical_region_id: RegionId,
+    region_dir: &str,
+) -> RegionCreateRequest {
+    let mut column_metadatas = vec![
+        ColumnMetadata {
+            column_id: 0,
+            semantic_type: SemanticType::Timestamp,
+            column_schema: ColumnSchema::new(
+                "greptime_timestamp",
+                ConcreteDataType::timestamp_millisecond_datatype(),
+                false,
+            ),
+        },
+        ColumnMetadata {
+            column_id: 1,
+            semantic_type: SemanticType::Field,
+            column_schema: ColumnSchema::new(
+                "greptime_value",
+                ConcreteDataType::float64_datatype(),
+                false,
+            ),
+        },
+    ];
+    for (i, tag) in tags.iter().enumerate() {
+        column_metadatas.push(ColumnMetadata {
+            column_id: i as u32,
```

Comment:
Looks like the column id is duplicated with `greptime_timestamp` and `greptime_value`

Reply:
create request to logical region doesn't needs column id. I've changed them to 0 to avoid confusion